.github/workflows/token.yml

```
@@ -0,0 +1,29 @@
+name: 'figma-tokens style-dictionary actions'
+on:
+  pull_request:
+
+jobs:
+  generate_scss:
+    name: generate_scss
+    runs-on: ubuntu-latest
+    # 'figma' という名前を含むブランチのみ動くように設定
+    if: contains(github.head_ref, 'figma')
+    steps:
```

```suggestion
    steps:
      - name: Get yarn cache directory path
        id: yarn-cache-dir-path
        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
      - uses: actions/cache@v3
        id: yarn-cache
        with:
          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
          restore-keys: |
            ${{ runner.os }}-yarn-
```

依存を取ってくる時、cacheを用いてください。
他のactionsで用いているのでそれを参考にしてください。